preload.js

```diff
@@ -51,7 +51,6 @@ window.getUserKeys = async () => {
 window.sessionFeatureFlags = {
   useOnionRequests: true,
   useTestNet: isTestNet() || isTestIntegration(),
-  useClosedGroupV2: false, // TODO DO NOT MERGE Remove after QA
   forceLegacyGroupsDeprecated: false, // TODO DO NOT MERGE Remove after QA
   useClosedGroupV2QAButtons: false, // TODO DO NOT MERGE Remove after QA
```

Comment:
Do we want to keep this feature? If so please remove the comment?

Reply:
If we merge after legacy groups are read-only then we should probably remove `forceLegacyGroupsDeprecated` too?

Reply:
yep, I'll keep this PR opened (and the comments) until I can fully remove it

Reply:
remoed